2019/PaperCall.md

```diff
@@ -49,25 +57,70 @@ We expect participants to follow these rules at conference and workshop venues a
 _Tell potential speakers what you want to hear. What kind of talks are you interested in hearing? What kind of topics? Who is your audience? Any details about the conference itself, your typical attendees, etc will be incredibly helpful to helping speakers formulate their talks._
 
 ```
-TODO
+Kotlin/Everywhere Minsk is expecting around 300 developers from Belarus and abroad.
+
+We are interested, but not limited in the following topics:
+
+* Software Development with Kotlin
```

Comment:
I think need to add Kotlin Multiplatform Projects and Coroutines

Reply:
Added.